force_wfmanager/left_side_pane/data_source_model_view.py

```diff
@@ -225,12 +225,15 @@ def _fill_slot_rows(self, input_slots, output_slots):
         """ Fill the tables rows according to input_slots and output_slots
         needed by the evaluator and the model slot values """
         available_variables = self._get_available_variables()
-
         input_representations = []
+
         for index, input_slot in enumerate(input_slots):
+            new_name = self.model.input_slot_info[index].name
             slot_representation = InputSlotRow(model=self.model, index=index)
+            new_name = (new_name if new_name in available_variables else '')
```

Comment:
Nit picky style thing: I'd just do this as
```
if new_name not in available_slots:
     new_name = ''
```
as I think it reads a little more clearly; either that or assign `slot_representation.name = (new_name if new_name in available_variables else '')` directly on L235 below.

Reply:
Also, I would probably group the lines dealing with the `new_name` variable together to avoid context switching when reading the code.